cmd/kubelet/app/options/options.go

```diff
@@ -34,9 +34,10 @@ import (
 type KubeletServer struct {
 	componentconfig.KubeletConfiguration
 
-	AuthPath      util.StringFlag // Deprecated -- use KubeConfig instead
-	KubeConfig    util.StringFlag
-	APIServerList []string
+	AuthPath          util.StringFlag // Deprecated -- use KubeConfig instead
+	KubeConfig        util.StringFlag
+	WaitForKubeConfig bool
```

Comment:
I'm confused about what this is doing.  Why wouldn't whatever is launching this wait until it had the information it needed.

If its managed in this file, why wouldn't we build an automatic interlock to avoid having an additional parameter?